README.md

```diff
@@ -22,45 +22,67 @@ spec:
 ```
 
 
-### Run Locally (OpenShift)
+### Install (OpenShift)
```

Comment:
@cnuland I would recommend that the beginning of the README provide an overview and the goals of what the operator is looking to accomplish. The section on "Adding Defined Quota Sizes to Cluster" seems out of place

Reply:
Cleaned it up some, added an overview section at the beginning and moved the overview of the quota below the installation